l10n_es_aeat_sii/__manifest__.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2017 Acysos - Ignacio Ibeas <ignacio@acysos.com>
+# Copyright 2017 Diagram Software S.L.
+# Copyright 2017 MINORISA - <ramon.guiu@minorisa.net>
+# Copyright 2017 Studio73 - Pablo Fuentes <pablo@studio73.es>
+# Copyright 2017 Studio73 - Jordi Tolsà <jordi@studio73.es>
+# Copyright 2017 Factor Libre - Ismael Calvo
+# Copyright 2017 Otherway - Pedro Rodríguez Gil
+# Copyright 2017-2020 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Javi Melendez <javimelex@gmail.com>
+# Copyright 2018 Angel Moya <angel.moya@pesol.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Suministro Inmediato de Información en el IVA",
+    "version": "13.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "https://odoospain.odoo.com",
```

Comment:
Please change URL to https://www.aeodoo.org/

Reply:
Maybe the right one is https://github.com/OCA/l10n-spain/

Reply:
Done!